pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx

```diff
@@ -90,14 +90,13 @@ export default function (props: GraphDashboardProps) {
         ))}
       </Axis>
     </LineGraph>,
-
     <LineGraph
       title="SQL Statement Contention"
       sources={nodeSources}
-      tooltip={`The total number of SQL statements that experienced contention ${tooltipSelection}.`}
+      tooltip={`A thirty-second moving average of the # of SQL statements executed per second that experienced contention ${tooltipSelection}.`}
```

Comment:
I actually got this description from the description above. Should I also change that to be number instead of #?
https://github.com/cockroachdb/cockroach/blob/1af116f6c9f56aafa117a5442fc8fa37b4c084da/pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx#L39

Reply:
The interval actually changes based on how much time has been selected the default 10min time gives a 10 second interval. Switching it to an hour makes it 30 second. I'm going to update it to remove the time since it changes.

Reply:
We are a bit inconsistent. I'd go with "number" everywhere - it's also used this way in our SQL Activity pages. Good catch on the interval!

Reply:
Agreed. We should spell out "number", thanks for tagging!